### PR TITLE
handle priority in WNS

### LINKED ISSUE #532: missed ability to override X-WNS-PRIORITY
Hello, from [microsoft documentation](https://docs.microsoft.com/en-us/windows/uwp/design/shell/tiles-and-notifications/wns-notification-priorities)

there a ability to set the priority on the notification request is done through an additional header on the POST request, X-WNS-PRIORITY. This is an integer value between 1 and 4.

Seems it is not possible to set it per push message. Maybe you can recommend at least a hack that can work, thanks

### PR DESCRIPTION
**Pull request by Fivell**
resolves #532

### CODE REVIEW

**Comment by Fivell** [comment]
@aried3r  please review

**Comment by Fivell** [comment]
@jhottenstein are you able to review ?

File: spec/unit/client/active_record/wns/raw_notification_spec.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@
     notif.app  = Rpush::Client::ActiveRecord::Wns::App.new(name: "aname")
     notif.uri  = 'https://db5.notify.windows.com/?token=TOKEN'
     notif.data = { foo: 'foo', bar: 'bar' }
+    notif.priority = 2
```
[the remark this answers is not in the document]
Comment (Fivell) [reply spec/unit/client/active_record/wns/raw_notification_spec.rb]: @aried3r  this  asserted indirectly with 
```expect(notification.valid?).to be(true) ```
I suggested there is same behaviour here is for uri and data parameter

**Comment by Fivell** [comment]
@aried3r  modified test and added expectation there is no X-WNS-PRIORITY key for header if no priority

**Comment by aried3r** [comment]
Looking good. I'll try to release a new version of rpush soon.

If you want to see it sooner, there's an issue up for grabs: #521 😅

**Pull request merged**